src/main/java/ch/uzh/ifi/hase/soprafs22/controller/LobbyController.java

```diff
@@ -135,15 +135,27 @@ public List<LobbyGetDTO> getLobby() {
         return lobbiesGetDTOs;
     }
 
-    @GetMapping("/{apiVersion}/game/match/{lobbyId}")
+    @PostMapping("/{apiVersion}/game/match/{lobbyId}")
     @ResponseStatus(HttpStatus.CREATED)
     @ResponseBody
+    public void createGame(@RequestHeader("token") String token, @PathVariable Long lobbyId) {
+
+        lobbyService.createGame(token, lobbyId);
+
+        // send message to client via socket
+        socketMessage.convertAndSend("/topic/lobby/" + lobbyId, "GameCreated");
```

Comment:
Is this the best place to have the socket connection and sending the message?
Wasn't it the LobbyController only for mapping? 
I am not sure, therefore I am asking.

Reply:
I don't know either, I think @luis-tm put the SimpMessagingTemplate in the lobbyController, so I just used it there. Would you put it into the lobbyService ?

Reply:
For me, the controller is the one which should communicate with the client, that why I included the socket connection there.

Reply:
Ah yes, that makes sense. I think I will then leave it then in the controller

Reply:
sounds good. So websocket related stuff --> controller.